### PR TITLE
Fix misc failing specs

### PR DESCRIPTION
**Pull request by josephbaylon**
#### Summary
Fix misc failing specs

#### Ticket Link
NA

### CODE REVIEW

File: e2e/cypress/integration/accessibility/accessibility_input_fields_spec.js [hunk 1]
```diff
@@ -111,6 +111,17 @@ describe('Verify Accessibility Support in different input fields', () => {
...
+        // # Adding at least five other users in the channel
+        cy.getCurrentChannelId().then((channelId) => {
+            cy.getCurrentTeamId().then((teamId) => {
+                cy.createNewUser({}, [teamId]).then((user) => {
+                    cy.apiAddUserToChannel(channelId, user.id);
+                });
+            });
+        });
+        cy.apiLogin('sysadmin');
+        cy.visit('/ad-1/channels/town-square');
```
Comment (saturninoabril) [inline e2e/cypress/integration/accessibility/accessibility_input_fields_spec.js]: Got these failing on this spec.
Screen Shot 2020-03-12 at 10 00 19 AM

File: e2e/cypress/integration/interactive_menu/basic_options_spec.js [hunk 2]
```diff
@@ -605,12 +605,12 @@ function verifyMessageAttachmentList(postId, isRhs, text) {
...
             and('have.css', 'width', '220px');
 
-        cy.findByPlaceholderText('Select an option...').
+        cy.findByPlaceholderText('Select an option...').scrollIntoView().
             and('have.css', 'height', '32px').
             and('have.css', 'width', '220px').
             and('have.css', 'padding-right', '30px');
 
-        return cy.findByPlaceholderText('Select an option...').invoke('attr', 'value').then((value) => {
+        return cy.findByPlaceholderText('Select an option...').scrollIntoView().invoke('attr', 'value').then((value) => {
```
Comment (saturninoabril) [inline e2e/cypress/integration/interactive_menu/basic_options_spec.js]: Got these failing
Screen Shot 2020-03-12 at 10 16 59 AM

File: e2e/cypress/integration/accessibility/accessibility_input_fields_spec.js [hunk 3]
```diff
@@ -111,6 +111,17 @@ describe('Verify Accessibility Support in different input fields', () => {
     });
 
     it('MM-22625 Verify Accessibility Support in Search Autocomplete', () => {
+        // # Adding at least five other users in the channel
```
[the remark this answers is not in the document]
Comment (josephbaylon) [reply e2e/cypress/integration/accessibility/accessibility_input_fields_spec.js]: My bad. I was undoing some code and didn't realize everything was undone. I put back the original intent

File: e2e/cypress/integration/accessibility/accessibility_input_fields_spec.js (hunk 1 above)
Comment (josephbaylon) [reply e2e/cypress/integration/accessibility/accessibility_input_fields_spec.js]: The second error isn't happening for me but I updated the way elements are retrieved to see if it passes for you this time

Comment (josephbaylon) [reply e2e/cypress/integration/accessibility/accessibility_input_fields_spec.js]: The first error is related to loop above

File: e2e/cypress/integration/interactive_menu/basic_options_spec.js (hunk 2 above)
Comment (josephbaylon) [reply e2e/cypress/integration/interactive_menu/basic_options_spec.js]: I'm not getting these failures but I added force:true to clear. Hopefully that fixes it

Comment (josephbaylon) [reply e2e/cypress/integration/interactive_menu/basic_options_spec.js]: Screen Shot 2020-03-12 at 9 40 48 AM

File: e2e/cypress/integration/accessibility/accessibility_input_fields_spec.js (hunk 1 above)
Comment (josephbaylon) [reply e2e/cypress/integration/accessibility/accessibility_input_fields_spec.js]: I also increased timeout for @ and ~, and made sure navigation arrows are what they are intended to do

Screen Shot 2020-03-12 at 10 51 13 AM

Comment (saturninoabril) [reply e2e/cypress/integration/accessibility/accessibility_input_fields_spec.js]: Thanks, all passed now.

File: e2e/cypress/integration/interactive_menu/basic_options_spec.js (hunk 2 above)
Comment (saturninoabril) [reply e2e/cypress/integration/interactive_menu/basic_options_spec.js]: Tested, all passed.

**Pull request merged**